### PR TITLE
Mobile Friendly

### PR DESCRIPTION
**Pull request by aradmargalit**
Our app looks bad on mobile. Not a big deal since upload from those devices isn't very likely, but no reason to look bad IMO.

**Current Production**
image
image
image
image

**This Branch**
image
image
image
image

### CODE REVIEW

File: src/components/GarminUploadInstructions/GarminUploadInstructions.tsx [hunk 1]
```diff
@@ -58,43 +59,43 @@ export function GarminUploadInstructions({
     setCookie(GARMIN_UPLOAD_INSTRUCTIONS_OPEN_COOKIE, newValue);
   };
 
+  const isMediumDisplayOrLarger = useDisplayIsSizeOrLarger('md');
+
   return (
-    <Stack spacing={1}>
```
Comment (aradmargalit) [inline src/components/GarminUploadInstructions/GarminUploadInstructions.tsx:62]: This outer stack only had one element inside it 👻

Comment (eshedmargalit) [reply src/components/GarminUploadInstructions/GarminUploadInstructions.tsx:62]: Humiliating...

File: src/components/Collapse/Collapse.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function Collapse({
   buttonAriaLabel,
 }: CollapseProps) {
   return (
-    <Stack direction="row" spacing={2}>
+    <Stack direction={{ md: 'row', xs: 'column' }} spacing={2}>
```
Comment (eshedmargalit) [inline src/components/Collapse/Collapse.tsx:25]: Didn't know you could do this, cool

File: src/components/StoredActivityIndicator/StoredActivityIndicator.tsx [hunk 3]
```diff
@@ -9,12 +9,14 @@ export function StoredActivityIndicator() {
     useGarminActivityStorage();
 
   const countString = `${storedActivities.length} Garmin ${
-    storedActivities.length > 1 ? 'activities' : 'activity'
+    storedActivities.length == 1 ? 'activity' : 'activities'
```
Comment (eshedmargalit) [inline src/components/StoredActivityIndicator/StoredActivityIndicator.tsx:12]: Why `==` and not `===`?

**Review by eshedmargalit — APPROVED** [review]
Nice

File: src/components/StoredActivityIndicator/StoredActivityIndicator.tsx (hunk 3 above)
Comment (aradmargalit) [reply src/components/StoredActivityIndicator/StoredActivityIndicator.tsx:12]: Typo, I'll fix

Comment (aradmargalit) [reply src/components/StoredActivityIndicator/StoredActivityIndicator.tsx:12]:
```suggestion
    storedActivities.length === 1 ? 'activity' : 'activities'
```

**Pull request merged**